conduit/data/datamodules/audio/base.py

```diff
@@ -0,0 +1,45 @@
+"""Base class for audio datasets."""
+from abc import abstractmethod
+from pathlib import Path
+from typing import Optional, Union
+
+import albumentations as A
+import attr
+from kit import implements
+
+from conduit.data.datamodules.base import CdtDataModule
+from conduit.data.datasets.utils import AlbumentationsTform
+from conduit.types import Stage
+
+__all__ = ["CdtAudioDataModule"]
+
+
+@attr.define(kw_only=True)
+class CdtAudioDataModule(CdtDataModule):
+
+    root: Union[str, Path] = attr.field(kw_only=False)
+
+    @property
+    @abstractmethod
+    def _base_augmentations(self) -> A.Compose:
```

Comment:
these augmentation properties were removed a couple of weeks ago and replaced with default_transforms, with the transforms now able to be freely specified through the init.

Reply:
That aside, the properties should return nn.Sequential objects, not A.Compose (which is image-specific).

Reply:
Aren't these my words?

Reply:
I have heard this before, is this essential or can it be done another time?

Reply:
You can just remove all the augmentation methods; they're not needed at the moment. _setup can also be removed as it isn't doing anything that the superclass's isn't.

Reply:
Done✔️